.circleci/config.yml

```diff
@@ -138,14 +138,14 @@ commands:
       - run: test -z `go fmt ./...`
       - run: make lint
       - run: make check
-      - run: make integration
       # Start a docker container and set TEST_PG to optimize running tests.
       - run: docker run -d --name some-postgres -p 5555:5432 -e POSTGRES_PASSWORD=pgpass -e POSTGRES_USER=pguser -e POSTGRES_DB=mydb postgres
       - run: echo 'export TEST_PG="host=localhost user=pguser password=pgpass dbname=mydb port=5555 sslmode=disable"' >> $BASH_ENV
       - run: echo 'export TEST_FLAG="-p 1"' >> $BASH_ENV
       - run:
           command: make test
           no_output_timeout: 15m
+      - run: make integration
```

Comment:
should we run make integration after make test(unit/integration tests)? I find that if make test is failing, make integration is likely failing, too. and this order will be more helpful for debugging since make test display exactly which unit tests are failing.

Reply:
Good idea :+1: